migrating/checklists/ossm-migrating-premigration-checklists-assembly.adoc

```diff
@@ -0,0 +1,257 @@
+// Module included in the following dir:
+//
+// * service-mesh-docs-main/checklist.adoc
+
+:_mod-docs-content-type: ASSEMBLY
+[id=ossm-migrating-premigration-checklists-assembly]
+= Premigration checklists
+include::_attributes/common-attributes.adoc[]
+:context: ossm-migrating-premigration-checklists-assembly
+
+toc::[]
+
+The checklists make the following assumptions:
+
+1. You have read "Important information to know before migrating" and understand the differences between {SMProduct} 2 and {SMProduct} 3.
+2. You have read through the migration recommendations.
+3. You want to migrate from {SMProduct} 2 to {SMProduct} 3.
+4. You are running {SMProduct} {SMv2Version}.
+5. You have upgraded your `ServiceMeshControlPlane` to the latest version.
+6. If you are using {KialiProduct}, you are running the latest version.
+7. You have installed the {SMProduct} Operator 3.
+
+To install the {SMProduct} 3 Operator, see: xref:../../install/ossm-installing-openshift-service-mesh.adoc[Installing OpenShift Service Mesh]
+
+[WARNING]
+====
+You must upgrade your {SMProduct} Operator 2 to the latest release **before** you install {SMProduct} Operator 3.
+
+If you upgrade {SMProduct} Operator 2 operator _after_ you install {SMProduct} Operator 3, you will need to uninstall and reinstall {SMProduct} Operator 3 to ensure the included custom resource definitions (CRDs) are up-to-date.
+====
+//Content is taken from here: https://github.com/openshift-service-mesh/sail-operator/blob/main/docs/ossm/ossm2-migration/README.md
+
+//check for updates at https://github.com/openshift-service-mesh/sail-operator/blob/main/docs/ossm/ossm2-migration/README.md
+
+== Checklist for configuring add on replacements in Service Mesh 2
+
+After you have installed {SMProduct} Operator 3, you need to disable and configure a replacement another option for the following add ons in your `ServiceMeshControlPlane` in {SMProduct} 2:
+
+* [ ] Disable network policy management: `spec.security.manageNetworkPolicy=false`. If you want to keep the Network Policies created by the 2.6 `ServiceMeshControlPlane`, you will need to recreate and manage these manually.
+* [ ] Disable Prometheus: `spec.addons.prometheus.enabled=false`
+** [ ] Configure the `ServiceMeshControlPlane` with OpenShift Monitoring as the replacement. These instructions also include installing a standalone `Kiali` resource. Both can be done at the same time. link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-integrating-with-user-workload-monitoring_observability[Integration user-workload monitoring]
+** [ ] Configure a replacement f you are using a different Prometheus solution. link:https://docs.redhat.com/en/documentation/red_hat_openshift_service_on_aws/4/html/service_mesh/[OpenShift AWS]
```

Comment:
```suggestion
** [ ] Configure a replacement if you are using a different Prometheus solution. link:https://docs.redhat.com/en/documentation/red_hat_openshift_service_on_aws/4/html/service_mesh/[OpenShift AWS]
```

Reply:
I don't think `OpenShift AWS` is the link/ref you want here. Maybe this one: https://docs.redhat.com/en/documentation/red_hat_openshift_service_on_aws/4/html/service_mesh/service-mesh-2-x#integration-with-external-prometheus-installation?

Reply:
You'd only do this step if you are *not* using the OpenShift monitoring configuration mentioned in the step above.